### PR TITLE
Assets page a11y fixes

### PR DESCRIPTION
**Pull request by fysheets**
### [EDUCATOR-2664](https://openedx.atlassian.net/browse/EDUCATOR-2664) [EDUCATOR-2665](https://openedx.atlassian.net/browse/EDUCATOR-2665) [EDUCATOR-2666](https://openedx.atlassian.net/browse/EDUCATOR-2666) [EDUCATOR-2667](https://openedx.atlassian.net/browse/EDUCATOR-2667) [EDUCATOR-2669](https://openedx.atlassian.net/browse/EDUCATOR-2669) [EDUCATOR-2670](https://openedx.atlassian.net/browse/EDUCATOR-2670) 

Quick fixes for Assets table A11y review

### CODE REVIEW

File: src/components/AssetsDropZone/index.jsx [hunk 1]
```diff
@@ -35,53 +35,60 @@ export default class AssetsDropZone extends React.Component {
...
+                </div>
+                <WrappedMessage
+                  message={messages.assetsDropZoneMaxFileSizeLabel}
+                  values={{
+                    maxFileSizeMB: this.props.maxFileSizeMB,
+                  }}
+                >
+                  { displayText => (
+                    <p aria-hidden id="asset-drop-zone-max-file-size-label" data-identifier="asset-drop-zone-max-file-size-label">
+                      {displayText}
```
Comment (thallada) [inline src/components/AssetsDropZone/index.jsx]: Why are we hiding the max file size label from screenreaders? Don't they need to know it too?

Comment (fysheets) [reply src/components/AssetsDropZone/index.jsx]: It's now a part of the aria-described by on the Browse button here: https://github.com/edx/studio-frontend/pull/179/files#diff-e5d111258070606d64e86dc6dbcd0a42R70

Comment (thallada) [reply src/components/AssetsDropZone/index.jsx]: Do screenreaders read out the contents of an element referenced by an `aria-describedby` even if that element has `aria-hidden`?

Comment (fysheets) [reply src/components/AssetsDropZone/index.jsx]: Yeah - I tested on Safari with Voiceover and it reads out:

Browse your computer, button
Maximum file size: 10 MB, you are currently on a button, ....

**Pull request merged**